### PR TITLE
Updates the wording of the yq v3 instructions

### PR DESCRIPTION
**Pull request by joeliedtke**
Wording tweaks to streamline and clarify the yq v3 instructions.

### CODE REVIEW

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-2448--competent-brattain-de2d6d.netlify.app/docs/gke/deploy/deploy-cli/
LGTM

File: content/en/docs/gke/deploy/management-setup.md [hunk 1]
```diff
@@ -51,7 +51,10 @@ For a more detailed explanation of the drastic changes happened in Kubeflow v1.1
 
     Then, to verify the installation, run `kustomize version`. You should see `Version:kustomize/vX.Y.Z` in the output if you've successfully deployed Kustomize.
 
-1. Install [yq](https://github.com/mikefarah/yq#install), and follow the official installation steps. Note: due to changes from [yq v3 to v4](https://mikefarah.gitbook.io/yq/upgrading-from-v3#navigating), Versions 4.x of yq will not work with Kubeflow. [Version 3.3.0](https://github.com/mikefarah/yq/releases/tag/3.3.0) is confirmed to work.
```
Comment (zijianjoy) [inline content/en/docs/gke/deploy/management-setup.md:54]: I think we should keep the following information:

```
Note: due to changes from [yq v3 to v4](https://mikefarah.gitbook.io/yq/upgrading-from-v3#navigating), Versions 4.x of yq will not work with Kubeflow. [Version 3.3.0](https://github.com/mikefarah/yq/releases/tag/3.3.0) is confirmed to work.
```

It provides a good context for where to look at if people want yq4 support later.

Comment (joeliedtke) [reply content/en/docs/gke/deploy/management-setup.md:54]: Ok, I've added that link back in. PTAL.

Comment (zijianjoy) [reply content/en/docs/gke/deploy/management-setup.md:54]: Thank you Joe!

**Comment by zijianjoy** [comment]
/lgtm
/approve

Thank you for the improvement!

**Pull request merged**